crystal_toolkit/apps/examples/migrationgraph.py

```diff
@@ -0,0 +1,34 @@
+import dash
+from dash import html
+from dash_mp_components import CrystalToolkitScene
+from monty.serialization import loadfn
+
+import crystal_toolkit.components as ctc
+from crystal_toolkit.settings import SETTINGS
+
+app = dash.Dash(assets_folder=SETTINGS.ASSETS_PATH)
+
+# create the MigrationGraph object
+mg = loadfn("LiMnP2O7_mg.json")
```

Comment:
Tried to run the example from the root dir with

```py
python crystal_toolkit/apps/examples/migrationgraph.py
```

and got the error

```
FileNotFoundError: [Errno 2] No such file or directory: 'LiMnP2O7_mg.json'
```

To make the example robust w.r.t. cwd, you might want to change

```diff
- mg = loadfn("LiMnP2O7_mg.json")

+ import os

+ module_dir = os.path.dirname(os.path.abspath(__file__))

+ mg = loadfn(f"{module_dir}/LiMnP2O7_mg.json")